test_unstructured/partition/test_text.py

```diff
@@ -50,6 +50,15 @@ def test_partition_text_from_file():
     assert elements == EXPECTED_OUTPUT
 
 
+def test_partition_text_from_bytes_file():
```

Comment:
this won't validate the original change [here](https://github.com/Unstructured-IO/unstructured/blob/e4944cdaf8cc570644f1a15da128701e6756bbe0/unstructured/partition/text.py#L55), right?

Reply:
Yes you're right - as you mentioned below I should be testing against the bytes file not the bytes string.